update-imported-docs/update-imported-docs.py

```diff
@@ -204,20 +211,27 @@ def main():
         # to catch when a user specifies a file that does not exist
         print("[Error] failed in loading config file - {}".format(str(ex)))
         return -2
-
+    
     os.chdir(root_dir)
 
     # create the temp work_dir
-    try:
-        print("Making temp work_dir")
-        work_dir = tempfile.mkdtemp(
-            dir='/tmp' if platform.system() == 'Darwin' else tempfile.gettempdir()
-        )
-    except OSError as ose:
-        print("[Error] Unable to create temp work_dir {}; error: {}"
-              .format(work_dir, ose))
-        return -2
-
+    if not is_running_inside_container():
+        print("Local environment detected, Making a temp work dir...")
+        try:
+            work_dir = tempfile.mkdtemp(
+                dir='/tmp' if platform.system() == 'Darwin' else tempfile.gettempdir()
+            )
+        except OSError as ose:
+            print("[Error] Unable to create temp work_dir {}; error: {}".format(work_dir, ose))
+            return -2
+    else:
+        print("Docker container detected, Using a Shared Volume")
+        if os.path.exists('/var/lib/docker/volumes/sharedVolume'):
+            work_dir = "/var/lib/docker/volumes/sharedVolume/_data"
```

Comment:
Should we assume the tool that runs containers is Docker? (Personally, I avoid this tool, but I know other people like it).